packages/utils/src/lib/filter/README.md

```diff
@@ -0,0 +1,176 @@
+# @acpaas-ui/ngx-components/utils
+
+Use the filter module to filter a set of data. The module provides easy to use filter components, a filter entitiy and a filter class.
+
+## Usage
+
+```typescript
+import { FilterModule } from '@acpaas-ui/ngx-components/utils'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### Filter components
+
+#### Checkbox filter
+
+##### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() filter: Filter;` | - | An instance of the Filter class. Its different options are explained in detail below. |
+| `@Output() update: EventEmitter<any>` | - | Will emit the selected value. |
+
+#### Input filter
+
+##### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() filter: Filter;` | - | An instance of the Filter class. Its different options are explained in detail below. |
+| `@Output() update: EventEmitter<any>` | - | Will emit the selected value. |
+
+#### Select filter
+
+##### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() filter: Filter;` | - | An instance of the Filter class. Its different options are explained in detail below. |
+| `@Output() update: EventEmitter<any>` | - | Will emit the selected value. |
+
+#### `Filter`
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `id: string;` | - | The filter id. |
+| `name: string;` | - | The filter name. |
+| `options: any[];` | - | An array of items which can be checked or selected. |
+| `value: string | any[];` | - | The value of the filter. |
+| `parse: (data: any[], value: any) => any[];` | - | The custom parsing that is used to filter the data. |
+| `parseData(data: any): any[];` | - | Basic parsing that is used to filter the data. This is used in the filter service. |
+
+### Example
+
+```typescript
+import { FilterModule } from '@acpaas-ui/ngx-components/utils';
+
+@NgModule({
+    imports: [
+        FilterModule
+    ]
+});
+
+export class AppModule {};
+```
+
+```typescript
+import { Filter, FilterService } from '@acpaas-ui/ngx-components/utils';
+
+    public searchFilter = new Filter();
+    public checkFilter = new Filter();
+    public selectFilter = new Filter();
+    public heroList = [
+        { id: 'id1', name: 'Batman' },
+        { id: 'id2', name: 'Wonder Woman' },
+        { id: 'id3', name: 'Wolverine' },
+        { id: 'id4', name: 'Iron Man' },
+        { id: 'id5', name: 'Deadpool' },
+    ];
+```
+
+```typescript
+constructor(public filterService: FilterService) {}
+```
+
+**Checkbox filter example**
```

Comment:
Maybe you should create a heading "Examples" with sub-headings, so it's consistent with the other READMEs.